requirements.txt

```diff
@@ -0,0 +1,2 @@
+flake8==2.5.4
```

Comment:
To reflect the change in the requirements.txt filename? OK, I'll update it. (But I plan to leave the rest of the README alone until we get the test suite to a workable state.)